specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/confidentialledger.json

```diff
@@ -141,7 +149,7 @@
           "200": {
             "description": "List of collections",
             "schema": {
-              "$ref": "#/definitions/CollectionArray"
+              "$ref": "#/definitions/PagedCollections"
```

Comment:
Same concern here. Don't create a new schema -- just add a `nextLink` property to `CollectionArray`.

Reply:
Thank you for your feedback! Does the fix I pushed look correct? @mikekistler